javascript/builtins/Intl.json

```
@@ -414,6 +414,56 @@
               "deprecated": false
             }
           },
+          "dateStyle": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "76"
+                },
+                "chrome_android": {
+                  "version_added": "76"
+                },
+                "edge": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "nodejs": {
+                  "version_added": null
+                },
+                "opera": {
+                  "version_added": null
+                },
+                "opera_android": {
+                  "version_added": null
```

There's no Opera Android out with Chrome "76", so we can set this to `false`.

```suggestion
                  "version_added": false
```